Automattic/studio
# Studio: Allow blocking user from adding demo sites

Related to https://github.com/Automattic/dotcom-forge/issues/7370

## Proposed Changes

This PR implements the UI for when the user is blocked from creating a demo site:

<img width="872" alt="Screenshot 2024-07-11 at 9 10 57 PM" src="https://github.com/user-attachments/assets/61de50d8-b708-43c8-85b1-e66b50271eca">

## Testing Instructions

* Navigate to the User Report Card for your own user
* Use the `Disable site creation` button (at the bottom of the User Report Card) to block your user from creating demo sites
* Pull the changes from this branch
* Start the app with `nvm use && npm install && npm start`
* Make sure to log in with the user that you just blocked from creating sites
* Navigate to `Share` tab
* Confirm that `Add demo site` button is disabled
* Hover over the button and confirm that it displays a message about demo sites being disabled for this account

## Pre-merge Checklist
